db/seeds.rb

```diff
@@ -1,3 +1,7 @@
 # frozen_string_literal: true
 
-User.create!(email: 'test@test.com', password: 'password')
+User.create!(email: 'test@test.com', password: 'password',
```

Comment:
I think some error because email will not pass uniqueness validation or...? How to edit this?

Why we need run db:seeds two times?)